api/controllers/StatsController.js

```diff
@@ -145,14 +145,17 @@ module.exports = {
         select: ['id', 'username'],
       });
       const currentSeasonMatches = Match.find({
-        startTime: { '>': currentSeason.startTime },
-        endTime: { '<': currentSeason.endTime },
+        startTime: { '>': dayjs(currentSeason.startTime).format('YYYY-MM-DD HH:mm:ss.SSS Z') },
```

Comment:
Silly question, but how does `currentSeason.startTime` compare to 

```
dayjs(currentSeason.startTime).format('YYYY-MM-DD HH:mm:ss.SSS Z');
```

Is `currentSeason.endTime` already a string in that format (if that's how postgres is storing timestampz columns? Is this equivalent to the follwing?
```suggestion
        startTime: { '>': currentSeason.startTime},
```

Reply:
So it should already be in that format, this is the format that postgres puts it in.